### PR TITLE
company logos, FAQs, content

### PR DESCRIPTION
**Pull request by yasmine0217**

### CODE REVIEW

**Comment by mrmcpowned** [comment]
Why were the logos deleted instead of simply removing the metadata from `src/config/data/sponsors.json`?

File: src/config/data/faq.js [hunk 1]
```diff
@@ -4,151 +4,98 @@ const liSpace = { marginBottom: "10px" };
...
+        registration form and we’ll make sure you get what you need.
       </>
     )
   },
   {
-    title: <>So, why ‘ShellHacks’?</>,
+    title: <>Will there be travel reimbersement?</>,
     description: (
       <>
-        Well, it started off as a joke about what a turtle-themed hackathon
```
Comment (Jehfkemsy) [inline src/config/data/faq.js]: what was wrong with this?

File: src/config/data/faq.js [hunk 2]
```diff
@@ -4,151 +4,98 @@ const liSpace = { marginBottom: "10px" };
...
-        <a
-          className="links"
-          rel="noopener noreferrer"
-          target="_blank"
-          href="https://fiu.edu"
-        >
-          FIU being no exception
-        </a>
-        .
+        ShellHacks will take place from September 20th through the 22nd at
```
Comment (Jehfkemsy) [inline src/config/data/faq.js]: Need to add a link for Kovens Center

File: src/pages/landing/supporters/index.jsx [hunk 3]
```diff
@@ -55,24 +55,11 @@ const Supporters = () => {
             />
           </a>
           <p className="landing-section-paragraph">
-            <a
```
Comment (Jehfkemsy) [inline src/pages/landing/supporters/index.jsx]: why did you remove the links for FIU and UPE?

File: src/assets/logos/sponsors/Redfin.svg [hunk 4]
```diff
@@ -1,20 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```
Comment (Jehfkemsy) [inline src/assets/logos/sponsors/Redfin.svg]: don't have to remove the actual image.... you can just remove them  from the jason file

File: src/config/data/faq.js (hunk 1 above)
Comment (yasmine0217) [reply src/config/data/faq.js]: Theres no need to have that FAQ anymore. They were getting long so I removed this

File: src/config/data/faq.js (hunk 2 above)
Comment (yasmine0217) [reply src/config/data/faq.js]: http://kovens.fiu.edu

Comment (yasmine0217) [reply src/config/data/faq.js]: I dont see a need to link the kovens center

File: src/pages/landing/supporters/index.jsx (hunk 3 above)
Comment (yasmine0217) [reply src/pages/landing/supporters/index.jsx]: Because they looked ugly and were unnecessary

File: src/assets/logos/sponsors/Redfin.svg (hunk 4 above)
Comment (yasmine0217) [reply src/assets/logos/sponsors/Redfin.svg]: I wanted to make sure that none of the images are accessible from a user using inspect element since the sponsors are a surprise
